### PR TITLE
Add darktooltip w/ git auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #10237 
Related issue(s): # #

@cdnjs/intern3 Please help me reviewing this PR. Thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/rubentd/darktooltip
 * Official website (optional, not the repository): http://rubentd.com/darktooltip
 * License and its reference: MIT, [ref](https://github.com/rubentd/darktooltip/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 7
   - Count of stars: 102
   - Count of forks: 24
# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by hare1039** [comment]
@pvnr0082t If a `.css` includes a picture file, should we host it?

**Comment by pvnr0082t** [comment]
@hare1039 Yes. we should include it.

**Comment by hare1039** [comment]
@sufuf3 I add the picture file. Because in `0.3.1/css/darktooltip.css` line 126: `'../img/modal-bg.png'`, I include the folders.
Please review this PR again. Thank you.

**Comment by hare1039** [comment]
@cdnjs/intern3 I amended the commit. Please help me reviewing this PR again. Thank you.

**Comment by PeterDaveHello** [comment]
Ping

File: ajax/libs/darktooltip/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/rubentd/darktooltip.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*darktooltip.@(js|css)",
+          "modal-bg.png"
```
Comment (pvnr0082t) [inline ajax/libs/darktooltip/package.json]: @hare1039 We should try to apply rules to exclude files we don't need under `dist` folder.

Comment (hare1039) [inline ajax/libs/darktooltip/package.json]: @pvnr0082t OK I change the style of the rules. Please review this PR again. Thanks

**Comment by hare1039** [comment]
@pvnr0082t Fixed the commit message. Please review this PR again. Thanks.

**Pull request merged**